andela/bp-esa-frontend
# #167220594: fix incorrect automation status in details modal

#### What does this PR do?
- This PR fixes incorrect automation status displayed in details modal.

#### Description of Task to be completed?
**actual**
- The details modal displays the automation status as a success even when some of the automation was not successful. 
**expected**
- The details model is only expected to display the automation status as a success only if only all the automation was successful.

#### How should this be manually tested?
1. Clone the repo, checkout to the `bg-card-success-response-167220594` and install all the dependencies.
2. run the server using this command `yarn start`
3. head over to the browser and paste this url `http://localhost:3000/`
4. click on the info icon on any of the cards, a modal with the automation details should show up showing success on if all the automation displayed in details are all successful otherwise it should display false.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[167220594](https://www.pivotaltracker.com/story/show/167220594)

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
![image](https://user-images.githubusercontent.com/25249904/61274425-788d7880-a7b4-11e9-810a-6858d3bea18b.png)
